CHANGELOG.md

```diff
@@ -196,19 +199,16 @@ this backend.
 - `@FILE, @LINE, @FN, @COLUMN` for debugging.
 
 
-
-
-
-
 ## V 0.1.18
 *16 Aug 2019*
 
-- Built-in ORM (`uk_customers = db.select from Customer where country == 'uk' && nr_orders > 0`)
-- Map initialization syntax: `m := { ‘foo’: ‘bar’, ‘baz’: ‘foo’ }`
+- Built-in ORM (`uk_customers = db.select from Customer where country == 'uk'
```

Comment:
I think it is better to not split the example code to 2 lines.

Reply:
Instead, the whole:
`uk_customers = db.select from Customer where country == 'uk' && nr_orders > 0`
can be on its own line